rust/storage/tests/object_storage_integration_test.rs

```diff
@@ -31,16 +42,26 @@ use chroma_storage::admissioncontrolleds3::AdmissionControlledS3Storage;
 use chroma_storage::config::{ObjectStorageConfig, ObjectStorageProvider};
 use chroma_storage::object_storage::ObjectStorage;
 use chroma_storage::Storage;
+use chroma_types::Cmek;
 
 // ============================================================================
 // CONFIGURATION - UPDATE THESE VALUES FOR YOUR ENVIRONMENT
 // ============================================================================
 
 /// GCS bucket name for testing
-const GCS_BUCKET_NAME: &str = "sandbox-sicheng";
+const GCS_BUCKET_NAME: &str = "storage-client-test";
 
 /// Test prefix to namespace all test objects
-const TEST_PREFIX: &str = "chroma-storage-test/";
+const TEST_PREFIX: &str = "object-store/";
+
+/// CMEK key resource name for testing
+/// Update this with your actual KMS key for CMEK tests
+/// Format: projects/PROJECT_ID/locations/LOCATION/keyRings/RING/cryptoKeys/KEY
+const GCS_CMEK_KEY_NAME: &str = GCS_INVALID_CMEK_KEY_NAME;
```

Comment:
Can we configure our github runners to actually run this test? Rather than only running it locally?